client/src/js/components/bhPatientFinancialActivity.js

```diff
@@ -59,6 +59,16 @@ function PatientFinancialActivityCtrl(Patients, moment, Session, Constants) {
       .finally(() => {
         $ctrl.loading = false;
       });
+
+    Patients.getStockMovements($ctrl.patientUuid)
```

Comment:
@lomamech, could you combine the `Patient.getFinancialActivity()` and `Patient.getStockMovements()` call with a `Promise.all()` and end it with the `.finally()`?

Like this:
```js
Promise.all([
 Patients.getFinancialActivity($ctrl.patientUuid),
 Patients.getStockMovements($ctrl.patientUuid),
]).then([financialData, stockData] => {
  // ... etc ...
})
.finally(() =>{
  $ctrl.loading = false;
});
```

The reason for doing this is that we don't want to set `$ctrl.loading =false` too early.